pkgs/applications/video/obs-studio/plugins/obs-webkitgtk.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, obs-studio
+, webkitgtk
+, glib-networking
+, meson
+, cmake
+, pkg-config
+, ninja
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obs-webkitgtk";
+  version = "unstable-2023-11-10";
+
+  src = fetchFromGitHub {
+    owner = "fzwoch";
+    repo = "obs-webkitgtk";
+    rev = version;
```

@fpletz should I still use the commit hash for the `fetchFromGitHub` invocation?

```suggestion
    rev = "ddf230852c3c338e69b248bdf453a0630f1298a7";
```

---

Oh, right! ;)